WalletWasabi.Fluent/Views/Wallets/Home/History/Details/TransactionDetailsView.axaml

```diff
@@ -82,7 +82,8 @@
       <!-- Confirmation Time -->
       <c:PreviewItem IsVisible="{Binding IsConfirmationTimeVisible}"
                      Icon="{StaticResource timer_regular}"
-                     Label="Confirmation time">
+                     Label="Expected confirmation time"
+                     ToolTip.Tip="This is an estimation based on current mempool statistics. The confirmation time might change.">
```

Comment:
[Bitcoin core's fee estimation](https://johnnewbery.com/an-intro-to-bitcoin-core-fee-estimation/) which WW uses is not based on the `current` mempool statistics, but we also consider current mempool data, so I would simplify that to:

```suggestion
                     ToolTip.Tip="This is just an estimation based on some data for transactions and blocks. The confirmation time might change.">
```